app/controllers/account/preferences_controller.rb

```diff
@@ -122,6 +116,10 @@ def update_copyright_holder(legal_name_change = nil)
     def prefs_changed_successfully
       result = false
       if !@user.changed
+        # NOTE: The next line appears to be unreachable
+        # because @user.changed is always truthy. (It's at least `[]`.)
+        # Perhaps `!@user.changed?` was intended, but it breaks tests.
+        # 2023-06-11 JDC
```

Comment:
@nimmolo: I tried to fix this but got (easily) confused. 
Any hints for how to about this. (It's something for another PR.)

@nimmolo, @pellaea:
I was also confused by the code around the other uncovered line. See lines 110-114.
I didn't understand why copyright holder is updated in PreferencesController. 
It doesn't seem to be something that users can change here (as opposed to AccountsController).
```ruby
    def update_copyright_holder(legal_name_change = nil)
      return unless legal_name_change

      Image.update_copyright_holder(*legal_name_change, @user)
    end
```

Reply:
@JoeCohen - No idea about `changed` vs `changed?`. What tests does it break? (Maybe the tests need fixing!)
Ditto for the `update_copyright_holder` question... I'll try to spin it up as soon as i get Vagrant working with Code again.

I think my "creativity" in this file was probably limited to breaking up the controller. I don't recall trying to change any of this functionality.

Reply:
@JoeCohen - It looks like `def update_copyright_holder` is only here because I accidentally left it here when splitting the file! Note that it's also defined in `Account::ProfileController` in exactly the same way. `legal_name_change` happens in `profile` not `preferences`.

`PreferencesController`'s line 20, within `def update`:  `update_copyright_holder(@user.legal_name_change)` is repeated in `ProfileController` line 91, within `def deal_with_possible_profile_changes`. 

In summary, I think you can safely delete anything to do with copyright holder name changes in the `PreferencesController` and its tests.

Reply:
@nimmolo: For another PR
`if @user.changed` is always truthy and therefore not useful as a test, so it should be something else perhaps `if !@user.changed?`
When I try `if !@user.changed?`, it breaks part of  Capybara `AccountTest#test_edit_preferences`,
in particular the stuff relating to change of email address.